frontend/src/features/mapSideBar/containers/MapSideBarContainer.test.tsx

```diff
@@ -149,7 +150,7 @@ describe('Parcel Detail MapSideBarContainer', () => {
 
         await wait(async () => {
           const editButton = await findByTestId('edit');
-          expect(editButton).toBeInTheDOM();
+          expect(editButton).toBeInTheDocument();
```

Comment:
What's the difference?

Reply:
Was curious too, but looked it up and I guess `toBeInTheDOM()` depreciated and was renamed to `toBeInTheDocument()` as the original name was misleading